config/src/config/network_config.rs

```diff
@@ -54,23 +48,21 @@ pub struct NetworkConfig {
     #[serde(skip)]
     pub seed_peers: SeedPeersConfig,
     pub seed_peers_file: PathBuf,
+    // network_keypairs contains the node's network keypairs.
```

Comment:
Should consider deleting redundant comments :P

Reply:
Done, thanks!